src/pymovements/utils/decorators.py

```diff
@@ -25,16 +25,16 @@
 from typing import Any
 
 
-def auto_str(cls):
+def auto_str(cls: Any) -> str:
     """
     Automatically generate __str__() to include all arguments. Can be used as a decorator.
     """
-    def shorten(value: Any):
+    def shorten(value: Any) -> str:
         if isinstance(value, float):
             value = f'{value:.2f}'
         return value
 
-    def __str__(self):
+    def __str__(self: Any) -> str:
```

Comment:
it's a bit strange seing `self ` being typed but is necessary as `auto_str` is a function not a class, right?

Reply:
yea :(